packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -301,3 +317,12 @@ export interface UtxosToAccountUTXO {
   txid: string
   vout: number
 }
+
+export interface AccountToAccountPayload {
+  [key: string]: string
+}
+
+export interface AccountToAccountUTXO {
+  txid: string
+  vout: number
+}
```

Comment:
You can actually just call this UTXO, it's a very common data structure.

```suggestion
export interface UTXO {
  txid: string
  vout: number
}
```

Reply:
Current there is a UtxosToAccountUTXO, you can merge into one `UTXO`